flink/v1.19/flink/src/test/java/org/apache/iceberg/flink/source/reader/TestArrayPoolDataIteratorBatcherRowData.java

```diff
@@ -107,7 +114,11 @@ public void testSingleFileLessThanOneFullBatch() throws Exception {
   public void testSingleFileWithMultipleBatches() throws Exception {
```

Comment:
I set some comments to `as(...)` and remove some explicit comments to avoid exceeding 150 lines per a method limitation which is defined in the checkStyle.